resources/Qtmodels/ErrorMessages.qml

```diff
@@ -0,0 +1,6 @@
+pragma Singleton
```

Comment:
Does this need to be a singleton? do we care about instantiation count? If not and if QML allows this I think we should remove this declarator? and possibly do the same for the other variables that are declared as singletons